web-client/src/presenter/actions/AdvancedSearch/getPractitionersByNameAction.ts

```diff
@@ -1,25 +1,38 @@
 import { state } from '@web-client/presenter/app.cerebral';
 
-/* gets practitioners matching the name
- *
- * @param {object} providers the providers object
- * @param {object} providers.applicationContext the application context
- * @param {Function} providers.get cerebral get function
- * @returns {object} contains the practitioners returned from the getPractitionersByNameInteractor use case
- */
 export const getPractitionersByNameAction = async ({
   applicationContext,
   get,
+  props,
+  store,
 }: ActionProps) => {
```

Comment:
Looks like we are missing the props type and return type here

Reply:
will do!

Reply:
Going to fix this one while testing.